ruby.test/syntax/lexical/identifier/IdentifierInvalid.spt

```diff
@@ -0,0 +1,54 @@
+module IdentifierInvalid
```

Comment:
What about a convention to call these files `<name>Test` or something?

Reply:
Good idea, will fix tomorrow or so